src/test/java/learn/field_agent/data/AgentJdbcTemplateRepositoryTest.java

```diff
@@ -50,7 +50,7 @@ void shouldFindHazel() {
         // aliases anywhere in the project, but when I run all tests in this file it passes. I've debugged this
         // several times but am failing to see why it won't retrieve the aliases.
         // It will always return them when I run the API request as well. At a total loss.
-        assertEquals(2, hazel.getAliases().size());
+        assertTrue(hazel.getAliases().size() >= 1);
```

Comment:
Since all of your repository unit tests are accessing the same test database, you need to use an assertion pattern like this so that your test can handle variations in the test data (as the order of tests can unexpectedly change).